src/Components/SavedReadings/SavedReadings.js

```diff
@@ -1,10 +1,70 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { cardImages } from '../../cardImageData'
 import './SavedReadings.css'
 
-function SavedReadings() {
-  return (
-    <h1>SavedReadings</h1>
-  );
+class SavedReadings extends Component {
+  constructor({ userSavedReadings }) {
+    super()
+    this.state = {
+      userSavedReadings: userSavedReadings
+    }
+  }
+
+  handleChange = (event) => {
+    const { id, value} = event.target
+    
+    this.setState(prevState => ({
+    userSavedReadings: prevState.userSavedReadings.map(item => {
+      const readingDate = Object.keys(item)
+      return (readingDate[0] === id ? Object.assign(item, { comments: value }) : item)
```

Comment:
This goes over my head. Not sure what Object.assign does but will definitely have to look it up!